From b54530cc71a14efd547b4731a6a6512d2c71df21 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 28 Nov 2016 11:09:29 +0800
Subject: [PATCH 1291/5242] MLK-13520-01 ARM: dts: Add 'timeout-sec' property
 for wdog node

commit  b621eaeddb52a47d5ac9ad2fd8a7f7a2c8020980 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add 'timeout-sec' property for wdog node of i.MX7ULP.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 056363c..640f69d 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -307,6 +307,12 @@
 			clocks = <&clks IMX7ULP_CLK_WDG1>;
 			assigned-clocks = <&clks IMX7ULP_CLK_WDG1>;
 			assigned-clocks-parents = <&clks IMX7ULP_CLK_FIRC>;
+			/*
+			 * As the 1KHz LPO clock rate is not trimed,the actually clock
+			 * is about 667Hz, so the init timeout 60s should set 40*1000
+			 * in the TOVAL register.
+			 */
+			timeout-sec = <40>;
 		};
 
 		wdog2: wdog@40430000 {
@@ -316,6 +322,7 @@
 			clocks = <&clks IMX7ULP_CLK_WDG2>;
 			assigned-clocks = <&clks IMX7ULP_CLK_WDG2>;
 			assigned-clocks-parents = <&clks IMX7ULP_CLK_FIRC>;
+			timeout-sec = <40>;
 		};
 
 		clks: scg1@403E0000 {
-- 
1.7.9.5

